packages/starryui-docs/pages/components/component-list.ts

```diff
@@ -18,4 +19,5 @@ export const componentList: StarryUIComponentDefinition[] = [
  themeDefinition,
  traitsDefinition,
  trayDefinition,
+ gridDefinition,
```

Comment:
🟨 Yes this is added correctly here to the docs and now I can see the example in the docs! Last thing here is to put the `grid` component in alphabetical order

Currently it's appearing at the end in the component list in the docs after `tray`

![Screenshot from 2023-10-29 12-11-57](https://github.com/starryui/starryui/assets/688553/b16822d0-acf3-451c-a77e-ccfb7d86dcf7)

Reply:
Done

Reply:
@Vikulakav 
❌ Line 22 `gridDefinition` needs to move up to become line 16, directly after `frameDefinition`

Reply:
✅ Perfect